src/event_packet.rs

```diff
@@ -48,16 +51,18 @@ impl TryFrom<&[u8]> for UsbMidiEventPacket {
     fn try_from(value: &[u8]) -> Result<Self, Self::Error> {
         let raw_cable_number = match value.get(0) {
             Some(byte) => *byte >> 4,
-            None => return Err(MidiPacketParsingError::MissingCableNumber)
+            None => return Err(MidiPacketParsingError::MissingHeader),
         };
 
-        let cable_number = CableNumber::try_from(u8::from(raw_cable_number)).expect("(u8 >> 4) < 16");
+        let cable_number =
```

Comment:
Maybe this should be handled as a ParseError/similar instead of just `expect`ing? If the MIDI packet is malformed for some reason this could reasonably happen, and straight up crashing in a `try_from` is probably not ideal. Maybe there should also be a sanity check here for the MIDI magic string (I assume that's what the first 4 bytes are?)

Reply:
0xFF >> 4 == 0xF which will never fail here, so maybe it should just be `.unwrap()` but I don't think we need a parse error